valor-software/ng2-bootstrap
# fix(build): fix window not defined on An Universal

Closes #5071

Steps for testing:
https://github.com/angular/universal-starter
npm i
ng update @angular/cli @angular/core
move current version of ngx-bootstrap(dist) to node_modules/ngx-bootstrap 
add datepicker to project
npm run build:ssr && npm run serve:ssr
Universal application on http://localhost:4000
No errors should be visible. (ERROR ReferenceError: window is not defined)


# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [ ] added/updated API documentation.
 - [ ] added/updated demos.
